addons/l10n_jo_edi/views/account_move_views.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_move_form" model="ir.ui.view">
+            <field name="name">account.move.form</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_move_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//sheet" position="before">
+                    <div class="alert alert-warning" role="alert"
+                        invisible="not l10n_jo_edi_error">
+                        <div class="p-0 m-0">
+                            <i class="fa fa-warning" role="img" title="EDI (Jordan)"/>
+                            <span class="mx-1">E-invoicing (Jordan) last attempt error</span>
+                        </div>
+                        <field name="l10n_jo_edi_error"/>
+                    </div>
+                </xpath>
+                <xpath expr="//group[@name='sale_info_group']" position="inside">
+                    <field name="l10n_jo_edi_state" invisible="not l10n_jo_edi_state"/>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="view_out_invoice_tree_inherit" model="ir.ui.view">
+            <field name="name">account.move.tree.inherit</field>
```

Comment:
why suddently putting an `inherit` here but not on all views? Choose a team.
Option 1:
```xml
        <record id="view_out_invoice_tree_inherit_l10n_jo_edi" model="ir.ui.view">
            <field name="name">account.move.tree.inherit.l10n_jo_edi</field>
```
Option 2:
```xml
        <record id="view_out_invoice_tree" model="ir.ui.view">
            <field name="name">account.move.tree</field>
```
... because the orm will already prefix your xml_id by `l10n_jo_edi`